index.html

```diff
@@ -1,59 +1,99 @@
-            </svg>
-                
-        </div>
-    </main>
-    
-</body>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Personal Portofolio</title>
+
+    <!-- favicon -->
+    <link rel="stylesheet" href="images/favicon-1.ico" type="image/x-icon" />
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+    <!-- Header -->
+    <header class="header">
+        <a href="" class="logo">john Doe</a>
+        <div class="nav-links">
+           <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+           </ul>
+        </div>
+        <span class="nav-menu-icon"><i class="fa-solid fa-bars"></i></span>
+    </header>
+    <div class="image">
+            <img src="./images/Ellipse18.png" alt="#">
+    </div>
+    <div class="image-2">
+           <img src="./images/Ellipse19.png" alt="#">
+    </div>
+    <main class="main">
+        <div class="main-intro">
+            <h1>Laura Jong</h1>
+            <h3>Software Developer</h3>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+
+        </div>
+        <div class="section-experience">
+            <div class="exp-card">
+                <h4>Language</h4>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+            <div class="exp-card">
+                <h4>Language</h4>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="exp-card">
+                <h4>Language</h4>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+
+        </div>

```

Comment:
Kindly use the proper `font-family` for this part of text to match the design.


| Your Design | Figma Design |
|-------------|---------------|
| <img width="372" alt="image" src="https://user-images.githubusercontent.com/34578631/171231263-eb2a2f37-9208-4060-affe-b6172605693d.png"> | <img width="370" alt="image" src="https://user-images.githubusercontent.com/34578631/171231315-1b12741c-305d-41bc-bdd9-4ec7d96673fd.png"> |